src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -51,29 +100,52 @@ def _make_request(self):
-            
-            query = query_pb2.Query()
-            if not hasattr(query, 'transactionGetRecord'):
-                raise AttributeError("Query object has no attribute 'transactionGetRecord'")
-            query.transactionGetRecord.CopyFrom(transaction_get_record)
-            
-            return query
-        except Exception as e:
-            print(f"Exception in _make_request: {e}")
-            raise
+        if not self.transaction_id:
+            raise ValueError("Transaction ID must be set before making the request.")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)